plugins/cassandra/cpp-driver

```diff
@@ -1 +1 @@
-Subproject commit 189e46e3386cc2be0fd7428c1072339b9c611d6e
+Subproject commit 019d3f62b39eb6ebf0a8605faa7130bd9863a9f9
```

Comment:
This change seems unrelated and is probably responsible for the CASS_USE_BOOST issues.